nemo/export/trt_llm/qnemo/tokenizer_utils.py

```diff
@@ -43,6 +46,12 @@ def get_nmt_tokenizer(nemo_checkpoint_path: str):
         tokenizer = SentencePieceTokenizer(
             model_path=os.path.join(nemo_checkpoint_path, tokenizer_cfg.model), legacy=legacy
         )
+    elif library == "tiktoken":
+        tmp_dir = tempfile.TemporaryDirectory()
+        tmp_path = os.path.join(tmp_dir.name, "vocab.json")
+        vocab_file = os.path.join(nemo_checkpoint_path, tokenizer_cfg.vocab_file)
+        shutil.copy(vocab_file, tmp_path)
+        tokenizer = TiktokenTokenizer(vocab_file=tmp_path)
```

Comment:
Could this be just `tokenizer = TiktokenTokenizer(vocab_file=vocab_file)` i.e. w/o creating `tmp_dir`?

Reply:
@janekl changed it